grakn-engine/src/main/java/ai/grakn/engine/controller/TasksController.java

```diff
@@ -65,32 +79,41 @@
 @Api(value = "/tasks", description = "Endpoints used to query and control queued background tasks.", produces = "application/json")
 public class TasksController {
 
+    private static final Logger LOG = LoggerFactory.getLogger(GraknVertexProgram.class);
```

Comment:
This logger should be for the current class. i.e. `private static final Logger LOG = LoggerFactory.getLogger(TasksController.class);`